src/PointwiseFunctions/AnalyticSolutions/Xcts/ConstantDensityStar.hpp

```diff
@@ -0,0 +1,172 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Prefixes.hpp"  // IWYU pragma: keep
+#include "DataStructures/Tensor/Tensor.hpp"     // IWYU pragma: keep
+#include "Elliptic/Systems/Xcts/Tags.hpp"       // IWYU pragma: keep
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+/// \cond
+class DataVector;
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace Xcts {
+namespace Solutions {
+
+/*!
+ * \brief A constant density star in general relativity
+ *
+ * \detail This solution describes a star with constant density \f$\rho_0\f$
+ * that extends to a (conformal) radius \f$R\f$. It solves the XCTS Hamiltonian
+ * constraint that reduces to the non-linear elliptic equation \f[
+ * \Delta^2\psi+2\pi\rho\psi^5=0
+ * \f] for the conformal factor \f$\psi\f$ (see `Xcts`) under the following
+ * assumptions \cite Baumgarte2006ug :
+ * - Time-symmetry \f$K_{ij}=0\f$
+ * - Conformal flatness \f$\overline{\gamma}=\delta\f$, so \f$\Delta\f$ is the
+ * flat-space Laplacian
+ * - Spherical symmetry
+ * Imposing boundary conditions \f[
```

Comment:
Is this a separate sentence from the previous line? When rendered they appear on the same line.